lib/ansible/modules/files/lineinfile.py

```diff
@@ -188,8 +188,8 @@
 - name: Ensure the JBoss memory settings are exactly as needed
   lineinfile:
     path: /opt/jboss-as/bin/standalone.conf
-    regexp: '^(.*)Xms(\\d+)m(.*)$'
-    line: '\1Xms${xms}m\3'
+    regexp: '^(.*)Xms(\d+)m(.*)$'
+    line: '\1Xms{{ xms }}m\3'
```

Comment:
This change doesn't' look correct since `standalone.conf` is expecting shell style variables, not Jinja style variables.

Reply:
Oh, perhaps I misunderstood the example then. I thought it was trying to put a value from an Ansible var (xms) in there rather than a literal reference to a shell var.